conf/quincy/upi/octo-5-node-env.yaml

```diff
@@ -0,0 +1,88 @@
+# Physical/Bare-metal environment required for RBD persistent write back cache.
```

Comment:
If needed Add this conf in pacific as well.

Reply:
@sunilangadi2 , This is next step after this PR merge. And for this activity, need Amar's help.